volunteer_base/wizard/__init__.py

```diff
@@ -0,0 +1,5 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+# -*- coding: utf-8 -*-
+
+from . import egagement_cancel
+from . import egagement_onhold
```

Comment:
Also fix typos in filenames

```suggestion
from . import engagement_cancel
from . import engagement_onhold
```